src/MetaGraphsNext.jl

```diff
@@ -321,6 +321,32 @@ end
 
 copy(g::T) where {T<:MetaGraph} = deepcopy(g)
 
+"""
+    code_for(meta::MetaGraph, vertex_label)
+
+Find the code for associated with a vertex with `vertex_label`. This can be useful to pass
+to methods inherited from `LightGraphs`. Note, however, that vertex codes could be
+reassigned after vertex deletion.
+
+```jldoctest
+julia> using MetaGraphsNext
+
+julia> using LightGraphs: Graph
+
+julia> meta = MetaGraph(Graph());
+
+julia> meta[:a] = nothing
+
+julia> code_for(meta, :a)
+1
```

Comment:
I don't understand what this is supposed to do. Why does this return `1`?

Reply:
Because vertex `:a` is at vertex number 1. Does that make sense? Could I update the docs to make it more clear?

Reply:
Where did you add the vertex? Did it automatically get added on the `setindex` call on L338?

Reply:
Yes

Reply:
Ok, well I guess I'll merge this. Let me know if I should add anything to the docs to clarify that vertices get automatically added.

Reply:
I think it's best if it were documented. Implicit mutation of a graph is a footgun.

Reply:
Ref https://github.com/JuliaGraphs/MetaGraphsNext.jl/commit/edd0b91a6ddfb7358e4d4d3781fd5511d27cf576